integration-tests/tests/client/process_blocks.rs

```diff
@@ -1558,6 +1558,30 @@ fn test_gc_after_state_sync() {
     assert!(env.clients[1].chain.clear_data(tries, 2).is_ok());
 }
 
+#[test]
+fn test_blocks_in_storage() {
+    let epoch_length = 1024;
+    let mut genesis = Genesis::test(vec!["test0".parse().unwrap(), "test1".parse().unwrap()], 1);
+    genesis.config.epoch_length = epoch_length;
+    let mut chain_genesis = ChainGenesis::test();
```

Comment:
shouldn't you change the `num_epoch_to_store` here in the test to a non-default value and test that?

Reply:
@mzhangmzz Oh, sorry. I just noticed that the test function wasn't updated yet. Please, take a look again.